client/landing/gutenboarding/onboarding-block/acquire-intent/index.tsx

```diff
@@ -56,6 +57,13 @@ const AcquireIntent: React.FunctionComponent = () => {
 		has_selected_site_title: hasSiteTitle(),
 	} ) );
 
+	const locale = useLocale();
+	React.useEffect( () => {
```

Comment:
Relatively minor, but can we put the `React.useEffect()` inside the config check?

Reply:
Nope! Did that first but linter complains about hooks needing to be consistently run, its probably safe (config won't change mid run) but it was easier to just do what it says.

Reply:
Decided to comment this out completely until CORS issues are resolved.